easel/lib/screens/creator_hub/creator_hub_screen.dart

```diff
@@ -389,8 +353,8 @@ class NFTsViewBuilder extends StatelessWidget {
 class BuildGridView extends StatelessWidget {
   final List<NFT> nftsList;
   final WidgetBuilder onEmptyList;
-
-  const BuildGridView({Key? key, required this.nftsList, required this.onEmptyList}) : super(key: key);
+  final CollectionType collectionType;
```

Comment:
I can still see collection type here.

Reply:
I have removed this the collection type variable, and committed code 12 hours ago.. I think I am unable to merge my this commit into pull request.


![image](https://user-images.githubusercontent.com/115459890/197837491-b7f6803e-d456-4d4d-bdb5-83ba7c2b19be.png)